lib/resources/dstu2/device.yaml

```diff
@@ -0,0 +1,141 @@
+---
+name: Device
+field_name_base_url: http://hl7.org/fhir/DSTU2/device-definitions.html#Device
+fields:
+- name: identifier
+  required: 'No'
+  cardinality: 0..*
+  type: Identifier
+  description: Identifier(s) from manufacturer, owner and others.
+  example: |
+    {
+      "identifier": [
+        {
+          "type": {
+            "coding": [
+              {
+                "system": "http://hl7.org/fhir/identifier-type",
+                "code": "SNO",
+                "display": "Serial Number"
+              }
+            ],
+          "text": "Serial Number"
+        },
+        "value": "SERIALNBRDA88ABA9-9FAD-4A03-9E9A-B52299AE9AE4"
+        }
+      ]
+    }
+  children:
+
+  - name: type
```

Comment:
This isn't linking to anything right now - I don't think this is intended to be a child of identifier, right?